newIDE/app/src/AssetStore/AssetStoreNavigator.js

```diff
@@ -13,6 +13,7 @@ export type AssetStorePageState = {|
   openedAssetShortHeader: ?AssetShortHeader,
   filtersState: FiltersState,
   ignoreTextualSearch: boolean,
+  scrollPosition?: ?number,
```

Comment:
Is this still needed?

Reply:
Yes, it's what stores the stack of scroll positions.